MODULE.bazel

```diff
@@ -27,6 +25,8 @@ go_deps.module(
 
 use_repo(go_deps, "com_github_bazelbuild_buildtools")
 
+### INTERNAL ONLY - lines after this are not included in the release packaging.
```

Comment:
I think that you can move the `### INTERNAL ONLY` header above the `go_deps = use_extension` line - that is only used to load buildifier for internal use within the bazel-skylib workspace.

Reply:
Done

Reply:
The Gazelle plugin requires `@com_github_bazelbuild_buildtools`, so I don't think it will build without this.
https://github.com/bazelbuild/bazel-skylib/blob/bc112d41fd71a5abf51ff827f19a6af9ed81af43/gazelle/bzl/BUILD#L19

Reply:
Didn't think of that, thanks for pointing it out.

Reply:
Moved :)